core/lib/zksync_core_leftovers/src/temp_config_store/mod.rs

```diff
@@ -5,8 +5,8 @@ use zksync_config::{
     configs::{
         api::{HealthCheckConfig, MerkleTreeApiConfig, Web3JsonRpcConfig},
         chain::{
-            CircuitBreakerConfig, MempoolConfig, NetworkConfig, OperationsManagerConfig,
-            StateKeeperConfig, TimestampAsserterConfig,
+            CircuitBreakerConfig, MempoolConfig, OperationsManagerConfig, StateKeeperConfig,
```

Comment:
BTW -- do we need `zksync_core_leftovers` at all now? IIRC we only needed it because `zkstack_cli` used to depend on `TempConfigStore`, and the rest of the stuff is easily removable.

Reply:
`TempConfigStore` [is also used](https://github.com/matter-labs/zksync-era/blob/a075b229f054aa21415ef8462b8ad07e4985e678/core/bin/zksync_server/src/main.rs#L182) in the `zksync_server` binary, and [`load_general_config` / `load_database_secrets`](https://github.com/matter-labs/zksync-era/blob/a075b229f054aa21415ef8462b8ad07e4985e678/core/lib/zksync_core_leftovers/src/temp_config_store/mod.rs#L212-L233) are used by most other binaries (e.g., [contract verifier](https://github.com/matter-labs/zksync-era/blob/a075b229f054aa21415ef8462b8ad07e4985e678/core/bin/contract-verifier/src/main.rs#L32)). Naturally, all of these would be removed after the new config system integration, so I'd suggest to wait for it.